rules-reviewed/quarkus/springboot/springboot-properties-to-quarkus.windup.xml

```diff
@@ -114,5 +114,55 @@
                 <matches pattern="(properties|yml|yaml)" />
             </where>
         </rule>
-     </rules>
+        <rule id="springboot-properties-to-quarkus-00005">
+            <when>
+                <filecontent filename="application{profile}.{extension}" pattern="springdoc.swagger-ui.path" />
+            </when>
+            <perform>
+                <hint title="Replace Spring Swagger endpoint mapping" effort="1" category-id="mandatory">
+                    <message>
+                        Replace `springdoc.swagger-ui.path` with `quarkus.swagger-ui.path`
+                        
+                        By adding `quarkus.swagger-ui.always-include=true` Quarkus will always expose the Swagger UI endpoint. 
+                        It is only exposed in Dev mode by default.
+                    </message>
+                    <link title="Quarkus Guide - using OpenAPI and Swagger" href="https://quarkus.io/guides/openapi-swaggerui" />
+                    <quickfix type="REPLACE" name="swagger-ui-replace">
+                        <replacement>quarkus.swagger-ui.path</replacement>
+                        <search>springdoc.swagger-ui.path</search>
+                    </quickfix>
+                </hint>
+            </perform>
+            <where param="profile">
+                <matches pattern=".*" />
+            </where>
+            <where param="extension">
+                <matches pattern="(properties|yml|yaml)" />
+            </where>
+        </rule>
+        <rule id="springboot-properties-to-quarkus-00006">
+            <when>
+                <filecontent filename="application{profile}.{extension}" pattern="springdoc.api-docs.path" />
+            </when>
+            <perform>
+                <hint title="Replace Spring OpenAPI endpoint mapping" effort="1" category-id="mandatory">
+                    <message>
+                        Replace `springdoc.api-docs.path` with `quarkus.smallrye-openapi.path`
```

Comment:
do we need to add something here to mention changing `=/openapi` to `=/swagger-ui`?

Reply:
Hi @m-brophy , 
in both cases the value on the right side of the is consistent, after moving to quarkus.
It is just the name of the property that needs to be changed.

This is the solution view of the file https://github.com/RedHat-Middleware-Workshops/spring-to-quarkus-todo/blob/solution/src/main/resources/application.properties

This is the original SB version of the file
https://github.com/RedHat-Middleware-Workshops/spring-to-quarkus-todo/blob/main/src/main/resources/application.properties